platform/platform_linux.cpp

```diff
@@ -144,8 +144,8 @@ Platform::Platform()
   }
   else
   {
-    string const devBuildWithSymlink = my::JoinPath(path, "..", "..", "data");
-    string const devBuildWithoutSymlink = my::JoinPath(path, "..", "..", "..", "omim", "data");
+    string const devBuildWithSymlink = my::JoinPath(path, "data");
+    string const devBuildWithoutSymlink = my::JoinPath(path, "..", "omim", "data");
```

Comment:
Shouldn't it be just `"..", "data"`?

Reply:
If you check the git repo out into "/somepath/omim" then the build script recommended for Linux will generate e.g. "/somepath/omim-build-debug/" and a symlinnk "/somepath/omim-build-debug/data -> /somepath/omim/data". The binary will be generated as " /somepath/omim-build-debug/MAPS.ME".
There will also be a "/somepath/omim-build-debug/MAPS.ME.app/Contents/Resources/", though that seems a bit like a build system bug.
These paths are relative to the binary, so if you do "..", "data", then it would be "/somepath/data", which does not exist and thus will not work.
"data" will get the symlink and work if that exists. "..", "omim" will work if the checkout has the standard name.
"MAPS.ME.app", "Contents", "Resources" should work as well, but seems just wrong for a Linux build.

Reply:
Indeed, I forgot. Thanks.